src/livekit/openIDSFU.ts

```diff
@@ -14,82 +14,78 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import { GroupCall, IOpenIDToken, MatrixClient } from "matrix-js-sdk";
+import { IOpenIDToken, MatrixClient } from "matrix-js-sdk";
 import { logger } from "matrix-js-sdk/src/logger";
+import { MatrixRTCSession } from "matrix-js-sdk/src/matrixrtc/MatrixRTCSession";
+import { useEffect, useState } from "react";
 
-import { Config } from "../config/Config";
+import { LivekitFocus } from "./LivekitFocus";
+import { useActiveFocus } from "../room/useActiveFocus";
 
 export interface SFUConfig {
   url: string;
   jwt: string;
 }
 
+export function sfuConfigEquals(a?: SFUConfig, b?: SFUConfig): boolean {
+  if (a === undefined && b === undefined) return true;
+  if (a === undefined || b === undefined) return false;
+
+  return a.jwt === b.jwt && a.url === b.url;
```

Comment:
```suggestion
  return a?.jwt === b?.jwt && a?.url === b?.url;
```

Why not ^?

Reply:
I find that much harder to understand, personally, and less obvious that it's doing the thing it intends to do. I'm finer with it, but I think it could use a comment, whereas I find the previous code a bit more self-documenting.

Reply:
I think that for me it's quicker to read my version as "are they equal" but I don't feel strongly about it. FWIW this would also work:

```suggestion
  if (a.url !== b.url) return false;
  if (a.jwt !== b.jwt) return false

  return true;
```

Reply:
Wouldn't that also need conditionals to avoid dereferencing `undefined`?

Reply:
I am not sure what you mean?

Reply:
The first thing it does is read `a.url` but `a` could be undefined in which case that will throw an exception.

Reply:
Ahm rug
```suggestion
  if (a?.url !== b?.url) return false;
  if (a?.jwt !== b?.jwt) return false

  return true;
```

Ah, right!

Reply:
OOB we've decided this doesn't really matter - resolving